test/integration/unprivileged_newproject_test.go

```diff
@@ -21,28 +22,6 @@ import (
 	templateclient "github.com/openshift/origin/pkg/template/generated/internalclientset"
 	testutil "github.com/openshift/origin/test/util"
 	testserver "github.com/openshift/origin/test/util/server"
-
-	// make sure all generated clients compile
-	// these are only here because it's the spot I chose to use a generated clientset for a test
-	"github.com/openshift/api/project"
```

Comment:
:tada: but that might be why integration has issues with missing types in scheme.